source/index.html

```diff
@@ -1,12 +1,147 @@
+              <p class="container-for-image-item-sedona">
+                <img class="sedona-image" src="../images/sedona-2.jpg" alt="Седона город" width="800" height="384">
+              </p>
+              <ul class="promo__list">
+                <li class="promo__item">
+                  <span class="hostel__logo"><span class="visually-hidden">Домик</span></span>
+                  <h4 class="advantages__title">Жильё</h4>
+                  <p class="advantages__description advantages__description--unwrapper">Рекомендуем пожить<br> в настоящем мотеле,<br> всё как в кино!</p>
+                </li>
+                <li class="promo__item">
+                  <span class="food__logo"><span class="visually-hidden">Еда</span></span>
+                  <h4 class="advantages__title">Еда</h4>
+                  <p class="advantages__description advantages__description--unwrapper">Всегда заказывайте<br> топовый фирменный бургер,<br> вы не разочаруетесь!</p>
+                </li>
+                <li class="promo__item">
+                  <span class="gift__logo"><span class="visually-hidden">Подарок</span></span>
+                  <h4 class="advantages__title">Сувениры</h4>
+                  <p class="advantages__description advantages__description--unwrapper">Не только китайского,<br>  но и настоящего местного производства!</p>
+                </li>
+              </ul>
+            </li>
+            <li class="advantages__item advantages__item--wrapper">
+              <p class="container-for-image-item-sedona">
+                <img class="sedona-image" src="../images/sedona-3.jpg" alt="Мост дьявола" width="800" height="384">
+              </p>
+              <div class="advantages-with-pic-description">
+                <h3 class="advantages__title white__title">Там есть<br> мост дьявола</h3>
+                <p class="advantages__description">Да, по нему можно пройти!<br> Если вы осмелитесь,<br> разумеется</p>
+              </div>
+            </li>
+            <li class="advantages__item advantages__item--unwrapper">
+              <h3 class="advantages__title">Небольшая<br> площадь</h3>
+              <p class="advantages__description advantages__description--unwrapper">Все достопримечательности<br> находятся очень близко</p>
+            </li>
+            <li class="advantages__item advantages__item--unwrapper">
+              <h3 class="advantages__title">Красивая<br> дорога</h3>
+              <p class="advantages__description advantages__description--unwrapper">Ехать в Седону из Лас-Вегаса<br> совсем не скучно!</p>
+            </li>
+            <li class="advantages__item advantages__item--unwrapper">
+              <h3 class="advantages__title">Мало<br> туристов</h3>
+              <p class="advantages__description advantages__description--unwrapper">Большинство едет в Гранд<br> Каньон и толпится там</p>
+            </li>
+          </ol>
+        </div>
+      </section>
+      <section class="searching_hostel">
```

Comment:
Если хотела задать как блок, то нужно использовать дефис
```suggestion
      <section class="searching-hostel">
```
т.к. нижнее подчеркивание это знак модификатора по БЭМ. (мы используем --, но _ тоже означает модификатор)